django/core/management/commands/test.py

```diff
@@ -47,6 +47,10 @@ def add_arguments(self, parser):
             help='Overrides the default address where the live server (used '
                  'with LiveServerTestCase) is expected to run from. The '
                  'default value is localhost:8081-8179.')
+        parser.add_argument('--tag', action='append', dest='tags',
```

Comment:
I remember considering that question before. The docs for custom test runners specify the signature of `run_tests` as taking `**kwargs`, I think specifically to avoid this type of problem. So I don't think we need to consider the addition of new kwargs as a back-compat break (though we could still mention it in the release notes, just in case).

That said, I don't see a problem with scoping this feature to `DiscoverRunner`. I guess it depends how likely it is that other custom runners might want to take advantage of this feature, too (and if they do, would they want to share our CLI options, or define their own).